agent/src/config.cc

```diff
@@ -121,7 +121,7 @@ config::config(const std::string& path) {
   _endpoint = json_config.get_string("endpoint");
 
   // pattern schema doesn't work so we do it ourselves
-  if (!RE2::FullMatch(_endpoint, "[\\w\\.:]+:\\w+")) {
+  if (!RE2::FullMatch(_endpoint, "[\\w\\.\\-:]+:\\w+")) {
     throw exceptions::msg_fmt(
         "bad format for endpoint {}, it must match to the regex: "
         "[\\w\\.:]+:\\w+",
```

Comment:
i think , we need to change also the message in the throw
```suggestion
        "[\\w\\.\\-:]+:\\w+",
```

Reply:
and also there is a typo "it must match to the regex" would be better as "it must match the regex"